README.md

```diff
@@ -60,6 +60,12 @@ For example, you can get all repositories from `http://192.168.56.1:8080/smeagol
 
 ### UI
 
+Make sure your SCM-Manager has the Smeagol Plugin installed!
+- https://192.168.56.2/scm/admin/plugins/available/
+
+If you use a local CES instance make sure the SCM-Manager CAS setting accept any proxy
```

Comment:
this is necessary if smeagol runs on the host machine

Reply:
I don't think I fully grasp your statement or question. But here goes:

In order to gain hot reload from a local development environment both SCM-Manager and CAS must run within a CES. But the SCMM must still be configured to accept any CAS proxying because otherwise CAS would reject the locally running Smeagol.

Does this help any further?

Reply:
- [X] done